src/applications/disability-benefits/new-686/config/chapters/chapter-0/1-wizard/wizard.md

```diff
@@ -0,0 +1,3 @@
+## Conceptual Overview
```

Comment:
I suspect this file should have some content, but I won't block the PR over it.

Reply:
It will have content in the future. Reading the ticket this branch is just for us to test out building a form from scratch. we are also exploring how we want to do documentation for the form so this file is just a scaffold we will fill in later.